### ISSUE TITLE
Feature request: Expose clean logs completely

### ISSUE THREAD

**Issue by Foxi352** [And3rsL/Deebotozmo#4]
For somewhere in the future when everything is working :-)

Why not expose clean log to be consumed by code using the lib. For now you just use the last clean image URL:
```
    def _handle_clean_logs(self, event):
        response = event['logs']
        self.last_clean_image = response[0]['imageUrl']
```
All dicts in response array contain more than just imageUrl. It contains for example a timestamp which could be interesting. Maybe store response in internal variable, and expose with a function like `getLogs(number: integer): dict` ?

**Comment by boriswerner** [comment]
Just for information: we have several reports that the log information coming from the ecovacs API is currently empty. Maybe they changed the API call. 
https://github.com/mrbungle64/ioBroker.ecovacs-deebot/issues/69
There are two calls (GetCleanLogs, GetLastCleanLog) which both do not contain information.

**Comment by And3rsL** [comment]
> Just for information: we have several reports that the log information coming from the ecovacs API is currently empty. Maybe they changed the API call.
> [mrbungle64/ioBroker.ecovacs-deebot#69](https://github.com/mrbungle64/ioBroker.ecovacs-deebot/issues/69)
> There are two calls (GetCleanLogs, GetLastCleanLog) which both do not contain information.

I think ecovacs API its heavly broken, it doesnt even show me the correct map into their app

![db2d58f4-ef38-4a5a-bcaf-8eccf5128646](https://user-images.githubusercontent.com/31848430/82918295-ce09e880-9f74-11ea-9eb6-969bf821b4b0.jpg)

**Comment by Foxi352** [comment]
My log information is working. I added a logging line to: 
```
def _handle_clean_logs(self, event):
    response = event['logs']
    _LOGGER.warning(response)
    self.last_clean_image = response[0]['imageUrl']
```
And here is the result (beautified) with some numbers replaced by XXX for security.
```
[
   {
      "ts":1590430581,
      "last":3051,
      "area":32,
      "id":"d3e7200e-XXXX-4784-ab21-27d299717698@1328823793@wilG",
      "imageUrl":"https://portal-eu.ecouser.net/api/lg/image/d3e7200e-XXXXXXXXX-ab21-27d299717698@1328823793@wilG",
      "type":"spotArea",
      "aiavoid":0,
      "aitypes":[

      ],
      "stopReason":1
   },
   {
      "ts":1590340198,
      "last":3180,
      "area":32,
      "id":"d3e7200e-1887-XXXX-ab21-27d299717698@1679377391@wilG",
      "imageUrl":"https://portal-eu.ecouser.net/api/lg/image/d3e7200e-XXXXXXXXX-ab21-27d299717698@1679377391@wilG",
      "type":"spotArea",
      "aiavoid":0,
      "aitypes":[

      ],
      "stopReason":1
   }
[... CONTINUES HERE ...]
]
```

**Comment by And3rsL** [comment]
> My log information is working. I added a logging line to:
> 
> ```
> def _handle_clean_logs(self, event):
>     response = event['logs']
>     _LOGGER.warning(response)
>     self.last_clean_image = response[0]['imageUrl']
> ```
> 
> And here is the result (beautified) with some numbers replaced by XXX for security.
> 
> ```
> [
>    {
>       "ts":1590430581,
>       "last":3051,
>       "area":32,
>       "id":"d3e7200e-XXXX-4784-ab21-27d299717698@1328823793@wilG",
>       "imageUrl":"https://portal-eu.ecouser.net/api/lg/image/d3e7200e-XXXXXXXXX-ab21-27d299717698@1328823793@wilG",
>       "type":"spotArea",
>       "aiavoid":0,
>       "aitypes":[
> 
>       ],
>       "stopReason":1
>    },
>    {
>       "ts":1590340198,
>       "last":3180,
>       "area":32,
>       "id":"d3e7200e-1887-XXXX-ab21-27d299717698@1679377391@wilG",
>       "imageUrl":"https://portal-eu.ecouser.net/api/lg/image/d3e7200e-XXXXXXXXX-ab21-27d299717698@1679377391@wilG",
>       "type":"spotArea",
>       "aiavoid":0,
>       "aitypes":[
> 
>       ],
>       "stopReason":1
>    }
> [... CONTINUES HERE ...]
> ]
> ```

Yes, API works to me too but images are totally messed up.. its yours ok?

**Comment by boriswerner** [comment]
It's a long known issue at ecovacs that the logs are incomplete or the images are messed up. For some its working better than for others. There is a nearly 30 pages long thread in a german forum (https://www.roboter-forum.com/index.php?thread/40009-reinigungsprotokoll-wird-nicht-gespeichert/). Ecovacs is doing "something" with nearly every update but wasnt able to solve it so far... For some (especially new users) the app shows logs but the API call is not returning anything so they even might be A/B testing something completely new...
My log images are most of the times also messed up.

**Comment by And3rsL** [comment]
Now https://github.com/And3rsL/Deebotozmo/blob/6887457b017fc322f5fae99ae043c8e8a593c3aa/deebotozmo/__init__.py#L234

contains api return
{'timestamp':cleanLog['ts'],'imageUrl': cleanLog['imageUrl'], 'type': cleanLog['type']}

**Comment by Foxi352** [comment]
I have last 20 logs in the `lastCleanLogs` array now, and it works perfectly ! All clean maps are working fine. Thank you for this ! Btw: i use lu / eu country / continent code ... And my account is only 2-3 weeks old.

BUT: You made __map a private property of the vacbot class. Since you moved the rooms[] into the Map class, there is now no more a clean way to fetch the room list from the lib.
